app/guid-file/template.hbs

```diff
@@ -1,281 +1,253 @@
-                </BsButton>
-            </h2>
+<OsfLayout @backgroundClass={{local-class 'bg-class'}} as |layout|>
+    <layout.main local-class='FileDetail_main'>
+        <div class='row'>
+            <div class='col-lg-11'>
+                <div local-class='FileDetail_registration'>
+                    <BsButton
+                        data-test-registration-link
+                        data-analytics-name='Registration'
+                        local-class='FileDetail__registration-link'
+                        @bubble={{true}}
+                        @type='link'
+                        @onClick={{'#'}}
+                    >
+                        <h5>
+                            {{this.model.file.provider}}
+                        </h5>
+                    </BsButton>
+                </div>
+            </div>
         </div>
-        <div data-analytics-scope='buttons' local-class='TitleBar__buttons'>
-            <div class='pull-right'>
-                <div class='btn-toolbar m-t-md'>
-                    {{#if this.canDelete}}
-                        <div class='btn-group m-l-xs m-t-xs'>
+        <div class='row' local-class='FileDetail_fileNameButtons'>
+            <div class='col-sm-10'>
+                <h3>
+                    <strong>
+                        {{this.model.file.name}}
+                    </strong>
+                </h3>
+            </div>
+            <div class='col-sm-1'>
+                <div class='btn btn-group'>
+                    <ResponsiveDropdown
+                        data-test-more-button
+                        data-analytics-scope='Files'
+                        as |dd|
+                    >
+                        <dd.trigger
+                            data-test-more-button-trigger
+                            data-analytics-name='Toggle more buttons'
+                            local-class='FileDetail_dropdown-trigger'
+                        >
+                            <Button
+                                local-class='FileDetail_dropdown-trigger-button'
+                            >
+                                <FaIcon @icon='ellipsis-v' @fixedWidth={{true}} />
+                            </Button>
+                        </dd.trigger>
+                        <dd.content
+                            local-class='FileDetail_dropdown-list'
+                        >
                             <BsButton
-                                data-analytics-name='Delete'
-                                data-test-delete-button
+                                data-test-download-button
+                                data-analytics-name='Download button'
                                 @bubble={{true}}
-                                @type='default'
-                                @size='sm'
-                                @onClick={{action (mut this.deleteModalOpen) true}}
+                                @onClick={{this.download}}
```

Comment:
Yes, here on the controller.ts file:

`@action
    download(version: number) {
        // To do: make this a link that looks like a button rather than calling this
        const url = `${this.downloadLink}?revision=${version}`;
        window.location.href = url;
    }`

Reply:
Yuhuai, after consulting with Brian this has been removed since a link should download the file.